src/lib/network/network_autoyast.rb

```diff
@@ -202,5 +186,40 @@ def ay_networking_section
 
       ay_profile["networking"]
     end
+
+    # Takes a list of udev rules and assignes them to corresponding devices.
+    #
+    # If a device has an udev rule defined already, it is overwritten by new one.
+    # Note: initialization of LanItems has to be done outside of this method
+    def assign_udevs_to_devs(udev_rules)
```

Comment:
It seems method `assign_udevs_to_devs` is not covered with unit tests. Is it tested in another way?